docs/spec_draft.md

```diff
@@ -1526,6 +1527,64 @@ operations correspond to [stablehlo.minimum](#stablehlominimum) and
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.collective_permute
+
+### Semantics
+
+Within each process group in the StableHLO grid, sends and receives data between
```

Comment:
Within each process group in the StableHLO grid, sends data from `source` process to `target` process for all `source_target_pairs`.

Reply:
Ok, I now see why we are using the same terminology. But `process_groups` give the impression of bi-directional data transfer.

Reply:
How about:

```
Within each process group in the StableHLO grid, sends the value of the 
`operand` tensor from the source process to the target process and produces a
`result` tensor.
```

Feedback:
  1. Good point about bidirectional data transfer. That was confusing to me as well.
  2. I think it's okay to use the terminology "source process" and "target process", emphasizing the relationship between these processes and values in `source_target_pairs`, even if these values aren't directly process ids.